frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/lambda/VerifyCodeHandler.java

```diff
@@ -263,7 +270,17 @@ private boolean checkErrorCountsForReauth(
                     ReauthAuthenticationAttemptsHelper.countTypesWhereUserIsBlockedForReauth(
                             countsByJourney, configurationService);
 
+            ClientRegistry client = userContext.getClient().orElse(null);
```

Comment:
What advantage does using orElse give if the client in the userContext is null?

Reply:
Rather than returning an optional we get null